specification/components/schemas/codings/CommentIsMandatory.yaml

```diff
@@ -0,0 +1,13 @@
+title: Comment is mandatory value
```

Comment:
Example values for properties are missing in all schemas under /schemas/codings/ but I don't think its a must have.

Reply:
I think examples are overkill if there is only one possible value